doc/source/getting_started/installation.rst

```diff
@@ -1,92 +1,99 @@
-.. _installation:
+.. _ref_installation:
 
 ============
 Installation
 ============
 
-The ``ansys-fluent-core`` package supports Python 3.9 through
-Python 3.12 on Windows and Linux.
 
-.. note::
-   This page provides information for quickly installing and launching
-   the ``ansys-fluent-core`` package. Additional PyFluent packages, which
-   are described later on this page, can also be installed. For
-   information on installing all PyFluent packages in a virtual environment,
-   see :ref:`faqs_install` in :ref:`faqs`.
+PyFluent Installation
+---------------------
 
-Install the package
--------------------
-Install the latest ``ansys-fluent-core`` package from
-`PyPi <https://pypi.org/project/ansys-fluent-core/>`_ with this code:
+PyFluent supports Python 3.9 through Python 3.12 on Windows, Mac OS and Linux.
 
-.. code::
+PyFluent can be installed, along with all its optional dependencies, using:
 
+.. code:: console
    pip install ansys-fluent-core
 
-If you plan on doing local *development* of PyFluent with Git, install the
-latest ``ansys-fluent-core`` package with this code:
+
+Development Installation
+------------------------
+The PyFluent source repository is available on GitHub. You can clone the repository and set up for local
+development with the following commands:
 
 .. code:: console
 
    git clone https://github.com/ansys/pyfluent.git
    cd pyfluent
    pip install pip -U
-   pip install -e .
-   python codegen/allapigen.py  # Generates the API files
+   pip install -e
+   python codegen/allapigen.py
 
+Step-by-Step Explanation
+~~~~~~~~~~~~~~~~~~~~~~~~
+1. Clone the Repository:
+
+.. code:: console
 
-Any changes that you make locally are reflected in your setup after you restart
-the Python kernel.
+   git clone https://github.com/ansys/pyfluent.git
+   cd pyfluent
 
-Launch Fluent
--------------
-To launch Fluent from PyFluent, use the :func:`launch_fluent() <ansys.fluent.core.launcher.launcher.launch_fluent>`
-method:
+- These commands clone the PyFluent repository from GitHub to your local machine and navigate into
+  the repository directory.
```

Comment:
I think it will look better without a bullet point behind it and without tabbing
```suggestion
These commands clone the PyFluent repository from GitHub to your local machine and navigate into
the repository directory.
```
See:
![image](https://github.com/ansys/pyfluent/assets/132297401/c0d57bd6-e0a1-4e29-9faa-3e870ac05b67)

Alternatively, can consider separating into subsections, first one being e.g.
```
Clone the Repository
++++++++++++++++++++
```